<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimePreviewUI</class>
 <widget class="QWidget" name="AnimePreviewUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>384</width>
    <height>169</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1,0">
   <item>
    <widget class="QLabel" name="coverImageLabel">
     <property name="text">
      <string>[Image]</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>[Title]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="specsLabel">
       <property name="text">
        <string>[Status, Dates, Episodes, Type, Genres]</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="descriptionLabel">
       <property name="text">
        <string>[Description]</string>
       </property>
       <property name="textFormat">
        <enum>Qt::RichText</enum>
       </property>
       <property name="wordWrap">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>[Tags]</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
